core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -376,24 +386,29 @@ private[spark] class CoarseMesosSchedulerBackend(
 
           val (afterCPUResources, cpuResourcesToUse) =
             partitionResources(resources, "cpus", taskCPUs)
-          val (resourcesLeft, memResourcesToUse) =
+          val (remainingMemResources, memResourcesToUse) =
```

Comment:
s/remainingMemResources/afterMemResources to be consistent w/ `afterCPUResources`